docs/artifactory/releasebundlecreate/help.go

```diff
@@ -0,0 +1,16 @@
+package releasebundlecreate
+
```

Comment:
Let's implement the changes suggested for this file, in all other release bundle help files.